test/models/pawn_test.rb

```diff
@@ -0,0 +1,148 @@
+		actual = @black_pawn.first_move?(4)
+		assert_equal expected_disallowed, actual  
+	end
+
+	test "can only move forward not backwards" do
+		pieces_to_remove = @game.pieces.where.not(:type => 'Pawn')
+		pieces_to_remove.update_all(:active => 0)
+
+ 		@white_pawn.update(:x_position => 5, :y_position => 2)
+ 		@black_pawn.update(:x_position => 5, :y_position => 7)
+
+		expected = true
+		# Move white pawn backwards
+		actual = @white_pawn.backwards_move?(1)
+		assert_equal expected, actual
+
+		# Move black pawn backwards
+		actual = @black_pawn.backwards_move?(8)
+		assert_equal expected, actual
+
+	end
+
+	test "cannot capture vertical/vertical obstacle" do 
+		pieces_to_remove = @game.pieces.where.not(:type => 'Pawn')
+		pieces_to_remove.update_all(:active => 0)
+
+ 		@white_pawn.update(:x_position => 5, :y_position => 5)
+ 		@black_pawn.update(:x_position => 5, :y_position => 6)
+ 		
+ 		expected = true
+ 		# Check for vertical obstacle for white pawn
+ 		actual = @white_pawn.vertical_obstacle?(6)
+ 		assert_equal expected, actual
+
+ 		# Check for vertical obstacle for black pawn
+ 		actual = @black_pawn.vertical_obstacle?(5)
+ 		assert_equal expected, actual
+	end
+
+	test "can only capture unfriendly diagonally" do 
+		pieces_to_remove = @game.pieces.where.not(:type => 'Pawn')
+		pieces_to_remove.update_all(:active => 0)
+		
+		@white_pawn.update(:x_position => 4, :y_position => 4)
+ 		@black_pawn.update(:x_position => 5, :y_position => 5)
+
+ 		expected = true
+ 		# Can capture black pawn diagonally
+ 		actual = @white_pawn.capture_diagonally?(5, 5)
+
+ 		assert_equal expected, actual
+
+ 		@black_pawn.update(:x_position => 6, :y_position => 6)
+ 		@white_pawn.update(:x_position => 5, :y_position => 5)
+
+ 		# Can capture white pawn diagonally
+ 		actual = @black_pawn.capture_diagonally?(5, 5)
+
+ 		assert_equal expected, actual
+
+ 		# Cannot capture a friendly diagonally
+ 		black_pawn_2 = @game.pieces.find_by(:type => "Pawn", :x_position => 7, :y_position => 7, :color => 0)
+ 		black_pawn_2.update(:y_position => 5)
+
+ 		actual = @black_pawn.capture_diagonally?(7, 5)
+ 		expected = false
+
+ 		assert_equal expected, actual
+ 	end
+
+ 	test "always allowed to move 1 forward if not blocked" do 
+ 		# Can always move forward 1 if no obstacles
+ 		pieces_to_remove = @game.pieces.where.not(:type => 'Pawn')
+		pieces_to_remove.update_all(:active => 0)
+
+		expected = true 
+		actual = @white_pawn.always_allowed(1,3)
+
+		assert_equal expected, actual
+
+		# Cannot move if change in x
+		expected = false
+		actual = @white_pawn.always_allowed(2,3)
+
+		assert_equal expected, actual
+	end
+
+	test "check all methods together within valid_move?" do
+		# Test first move up 2 but x changes by 1
+		expected = false 
+		actual = @white_pawn.valid_move?(4,2)
+
+		assert_equal expected, actual
+
+		# Test 1st move 1 forward but met by obstacle
+		@black_pawn.update(:y_position => 4)
+		actual = @white_pawn.valid_move?(4,1)
```

Comment:
@leamerluck I haven't checked all the other tests yet; but on line 138, it seems like the arguments for valid_move are invalid (a pawn with location—1, 2—can't move to the right). Did you mean `valid_move?(1,4)`? 
And if that's the case, you're actually moving the white pawn by 2 squares, not 1. So you might need another test for when a pawn moves 1 square but is blocked.